packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -130,11 +130,11 @@ function Navbar({
               testId="store-input-mobile"
               buttonTestId="store-input-mobile-button"
               onSearchClick={handlerExpandSearch}
+              hidden={!searchExpanded}
+              aria-hidden={!searchExpanded}
             />
 
-            <Suspense fallback={<ButtonSignInFallback />}>
```

Comment:
Maybe in the future we can revisit this logic and test the dynamic import, since the `ButtonSignIn` is not used in mobile (just inside the NavbarSlider, not here) and LH run the tests in Moto G4, so we don't need to load its javascript for perf sake. Not so important for now, though.
 
ps: we need to remove the import of `ButtonSignInFallback` then.